src/main/scala/uk/gov/nationalarchives/FileProcessor.scala

```diff
@@ -61,16 +61,26 @@ class FileProcessor(
       fileInfo: FileInfo,
       metadataFileInfo: FileInfo,
       cite: String,
+      name: Option[String],
```

Comment:
It's not just the folder name because it's used for the asset as well. It's actually the name of the judgment from the parser. I'll call it `judgmentName`